include/af/defines.h

```diff
@@ -402,14 +402,14 @@ typedef enum {
 
 #if AF_API_VERSION >=34
 typedef enum {
-    AF_RANDOM_PHILOX_4X32_10 = 100,
-    AF_RANDOM_THREEFRY_2X32_16 = 200,
-    AF_RANDOM_MERSENNE_GP11213 = 300,
-    AF_RANDOM_PHILOX = AF_RANDOM_PHILOX_4X32_10,
-    AF_RANDOM_THREEFRY = AF_RANDOM_THREEFRY_2X32_16,
-    AF_RANDOM_MERSENNE = AF_RANDOM_MERSENNE_GP11213,
-    AF_RANDOM_DEFAULT = AF_RANDOM_PHILOX
-} af_random_type;
+    AF_RANDOM_ENGINE_PHILOX_4X32_10 = 100,
+    AF_RANDOM_ENGINE_THREEFRY_2X32_16 = 200,
+    AF_RANDOM_ENGINE_MERSENNE_GP11213 = 300,
+    AF_RANDOM_ENGINE_PHILOX = AF_RANDOM_ENGINE_PHILOX_4X32_10,
+    AF_RANDOM_ENGINE_THREEFRY = AF_RANDOM_ENGINE_THREEFRY_2X32_16,
+    AF_RANDOM_ENGINE_MERSENNE = AF_RANDOM_ENGINE_MERSENNE_GP11213,
+    AF_RANDOM_ENGINE_DEFAULT = AF_RANDOM_ENGINE_PHILOX
+} af_random_engine_type;
```

Comment:
Can you align these by **=** and add documentation comments about what they are. You can look at this for reference https://github.com/arrayfire/arrayfire/blob/devel/include/af/defines.h#L304-L317